### ISSUE TITLE
Formal RFC: Pryaxis as a Moral Authority on Terraria

### ISSUE THREAD

**Issue by Ijwu** [Pryaxis/handbook#3]
I would like to put forward a set of guidelines which outline Pryaxis' involvement in matters of ethics or morality in the TShock and Terraria communities.

The main points I would like solidified in the handbook are:

* Pryaxis is **not** a moral authority on Terraria.
    * Pryaxis can not and will not take action on immoral or unethical actions within the Terraria game not hosted on Pryaxis infrastructure.
* Anything which does not happen on Pryaxis infrastructure or community pages is not subject to judgment or action from Pryaxis.
* Immoral or unethical actions are rated as such based on judgments from Pryaxis leadership as well as applicable laws in the United States of America. 
* Pryaxis is headquartered in Colorado and is subject to the laws within this state.
* Discord is created by Hammer and Chisel Inc., which is HQ'd in San Francisco and is subject to California laws.
    * Immoral or unethical behavior which occurs within the TShock Discord community will be dealt with by Pryaxis leadership.
    * Immoral or unethical behavior which occurs in DM on Discord should be reported to the Discord Administration team. This is out of the jurisdiction of Pryaxis.
* Extremely immoral or unethical behavior may be escalated to external authorities as Pryaxis leadership sees fit.

TL;DR is that if it didn't happen in our community or on our servers: we don't care. Take care of your issues with proper authorities but don't come to us for validation of your judgments. We are not the police nor do we have the capability to act as such. We will act on all unethical or immoral behavior which we have witnessed within our communities or infrastructure. We will escalate immoral or unethical behavior within our communities or infrastructure as we see fit.

**Comment by hakusaro** [comment]
I would actually tend to disagree with the scope but not the intent of this particular RFC. I think in some cases, like in #2, we can make the _decision_ to hold our community to standards that we deem appropriate. This RFC stemmed out of an issue that happened earlier which was primarily a case of "remote law enforcement," and I tend to agree that it's not our business to enforce things that leak into our community because people think we're the authority on everything.

I would say that, where ethics are concerned, it's too much of a grey area. I submitted #2 out of the desire to not have to support people who were running for profit businesses based on stealing other people's work. This is a flat out no go zone for me. Copyright infringement for private use has its own moral arguments separate from commercial profit. In this case, my suggestion was based out of the egregious violation of trust and security that server owners come to expect from the TShock Team & Pryaxis.

Other moral/legal zones are different -- in the case we received today, it was most assuredly:

- Presented in a way that couldn't authenticate the accusations that happened.
- Isolated from Pryaxis as the source.
- In an area where we have no legal or moral obligation policing.

In this regard, I think we shouldn't act. In other cases, though, I think we're obligated. It's not like RL is doing anything about this stuff. As a community, however, we have the capability to do something.

> What needs to be done is done by those capable.

**Comment by hakusaro** [comment]
E.g., Let's say some crazy person is a nazi in every discord but ours. He's a verified accredited nazi to literally all jews in literally all discords but ours. If he shows up, should we not ban him?

**Comment by Ijwu** [comment]
The conversation spurred by #2 happened _because_ the incident occurred in our community. I cannot say, with good faith, that we should act as moral authority for any scope outside of our own. It seems to presumptuous of us. I can see where #2 may conflict with this RFC. I think at that point we should work to figure out what the best solution is through leadership discourse. If we do make moral decisions then we should do so with a vast majority of the leadership team in agreement.

**Comment by hakusaro** [comment]
I think part of it did, but if we take action on, say, Voldey, then we're going to be breaking that line because technically it isn't in our community (re. #2).

**Comment by Ijwu** [comment]
I will form a PR with a proper document written up taking this issue into account. I will not be editing the issue I originally posted. That will allow us to preserve history and incorporate all feedback into the final product.

**Comment by hakusaro** [comment]
You could just post a new draft here.

**Comment by Ijwu** [comment]
That's smarter. Will do.

**Comment by Ijwu** [comment]
Here are the updated points including the conversation between myself and @hakusaro. Changes are bolded.

* Pryaxis is **not** a moral authority on Terraria.
    * Pryaxis can not and will not take action on immoral or unethical actions within the Terraria game not hosted on Pryaxis infrastructure.
* **Pryaxis will not be an accomplice to or allow for users who violate our moral or ethical values.**
    * **i.e. If a user violates our ethical standards outside of our infrastructure we can still hold them accountable to that fact.**
* ~Anything which does not happen on Pryaxis infrastructure or community pages is not subject to judgment or action from Pryaxis.~
* Immoral or unethical actions are rated as such based on judgments from Pryaxis leadership as well as applicable laws in the United States of America. 
* Pryaxis is headquartered in Colorado and is subject to the laws within this state.
* Discord is created by Hammer and Chisel Inc., which is HQ'd in San Francisco and is subject to California laws.
    * Immoral or unethical behavior which occurs within the TShock Discord community will be dealt with by Pryaxis leadership.
    * Immoral or unethical behavior which occurs in DM on Discord should be reported to the Discord Administration team. This is out of the jurisdiction of Pryaxis.
* Extremely immoral or unethical behavior may be escalated to external authorities as Pryaxis leadership sees fit.

**Comment by hakusaro** [comment]
> i.e. If a user violates our ethical standards outside of our infrastructure we can still hold them accountable to that fact.

Looks good to me and this comment definitely wasn't edited.

**Comment by Ijwu** [comment]
@hakusaro Naw, I put `i.e.` because I intended to provide further clarification, not an example. :)

**Comment by hakusaro** [comment]
@Ijwu get back on topic I don't know what you're talking about

**Comment by Ijwu** [comment]
@Pryaxis Look good to the rest of you? I'm being baited by Shank here into non-topic conversations.

**Comment by hakusaro** [comment]
@Ijwu you forgot "sloppily" and "without logic" in this sentence:

> Discord is created by Hammer and Chisel Inc., which is HQ'd in San Francisco and is subject to California laws.

**Comment by QuiCM** [comment]
LGTM

**Comment by Ijwu** [comment]
I've added assignees from whom I'd appreciate their opinions. Please give me a looksie guys and once that's done I can PR an official document in. Pls&ThankYou

**Comment by hakusaro** [comment]
@Ijwu 

1. Set a deadline on the request (say a week).
2. Assume anyone who hasn't said anything is not part of the consensus.
3. Write up the final report and move forward with it.

If people don't voice their opinions on it, you can't make them. If they're stalling the process that's their own damn fault.

**Comment by hakusaro** [comment]
(In which case, you can skip them)

**Comment by Ijwu** [comment]
@hakusaro The intention was to inform anyone who was waiting for another draft before commenting. I was going to move on in a few days anyway.

Edit: Well. Probably move on. Maybe after some badgering in Discord. :)

**Comment by ivanbiljan** [comment]
Lgtm

**Comment by tylerjwatson** [comment]
Hi guys. 

Respectfully, I don't agree with any part of this handbook, so you probably won't get the best answers on part of myself. It's best if you remove me from the handbook stuff.

Cheers!

**Comment by Patrikkk-zz** [comment]
I agreed with the first version more.

"Anything which does not happen on Pryaxis infrastructure or community pages is not subject to judgment or action from Pryaxis."

I know it is irrevelant how I manage my own yard. But basically if there is a member in my community who has history of misbehaving in other communities, but follows the rules of my community, i let them be. As long as they don't violate any of my rules.
Just thought it's fair.

If they are really a shitty person they will violate rules sooner or later. But they do get the chance to stay in the community.
So yeah no thumbs up from me yet :)

**Comment by hakusaro** [comment]
> But basically if there is a member in my community who has history of misbehaving in other communities, but follows the rules of my community, i let them be. As long as they don't violate any of my rules.

That's really fair, but I would actually question:

1. What happens if a person joins with the intent of being _exploitative_ of this policy?

Since piracy is a good example, let's use that. Let's assume a known pirate joins the community. He's pirated 30 or so plugins from other servers via malicious cracking. If he or she joins with the intent of advertising their community full of stolen plugins, we let them? Based on what you said, as long as they haven't broken any rules explicitly in our community, they should stay.

**Comment by bartico6** [comment]
Then reserve ~~a~~ the right to remove users whose previous behaviour has indicated an intent to violate the rule sets in place on Pryaxis' infrastructure, then.   
  
In short words, if someone's being disruptive everywhere they appear and they appear in your server *and* you have good faith that they will be disruptive here too, then you should have the right to remove them to save yourself moderating time.  
  
In other words, it's still your Discord/community and you're the ones in charge, if someone attempts to use "clever logic" to find loopholes just to get around your rules to anger you, then they're about as useful to the community as someone who got rightfully banned. In fact, if someone joins with the intent of being exploitative of your policies, that falls under your "violates our moral standards" rule, so you're clear on that.

**Comment by QuiCM** [comment]
A better phrasing could be
>Anything which does not happen on Pryaxis infrastructure or community pages **may be** subject to judgment or action from Pryaxis, **at our discretion**.

i.e., we can choose whether or not we want to act, on a case-by-case basis

Which is pretty much what @bartico6 said, just more concisely

**Comment by bartico6** [comment]
Correct. If you're setting up rules that are going to define what you can or cannot do, you should leave some sort of "hole" to use for extraordinary situations. I know this sounds shady, but in reality you should be able to make decisions without having to wait for everyone to push a rule change. Since you're making changes now, you might as well make your life easier for later.

**Comment by Patrikkk-zz** [comment]
I'm down for case-by-case basis
